lte/gateway/python/integ_tests/s1aptests/test_paging_with_mme_restart.py

```diff
@@ -86,10 +86,7 @@ def test_paging_with_mme_restart(self):
             ue_cntxt_rel_req,
         )
         response = self._s1ap_wrapper.s1_util.get_response()
-        self.assertEqual(
-            response.msg_type,
-            s1ap_types.tfwCmd.UE_CTX_REL_IND.value,
-        )
+        assert response.msg_type == s1ap_types.tfwCmd.UE_CTX_REL_IND.value
```

Comment:
The similar changes are required in other test cases, test_attach_detach_secondary_pdn_no_disconnect.py, test_attach_detach_multiple_secondary_pdn.py, test_attach_implicit_detach_timer_expiry.py, test_enb_partial_reset_multi_ue.py, test_enb_partial_reset_with_unknown_ue_s1ap_ids.py, test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn.py and many more.

Reply:
Yes, I am aware that about 100 tests or so still use `self.assertEqual` and similar functions instead of `assert`. The goal of this PR was to just fix the incorret `assertTrue`, where an `assertEqual` should have been used. A switch from the old to the new syntax in all tests will be done in a separate PR (maybe these changes here should be moved to that PR as well then?).

Reply:
nvm, I added these changes and separated them into several commits by fixing the incorrect asserts and changing the syntax